script.module.neverwise/addon.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.neverwise" name="NeverWise Kodi toolkit" version="1.0.9" provider-name="NeverWise">
+<addon id="script.module.neverwise" name="NeverWise Kodi toolkit" version="1.1.1" provider-name="NeverWise">
   <requires>
     <import addon="xbmc.python" version="2.14.0"/>
-    <import addon="script.module.beautifulsoup" version="3.2.1"/>
+    <import addon="script.module.beautifulsoup4" version="4.3.2"/>
+    <import addon="script.module.dateutil" version="2.4.2"/>
   </requires>
   <extension library="lib" point="xbmc.python.module"/>
   <extension point="xbmc.addon.metadata">
     <platform>all</platform>
-    <summary lang="it">Strumenti per gli addons audio/video di Kodi.</summary>
-    <summary lang="en">Toolkit for Kodi audio/video addons.</summary>
-    <description lang="it">Questi strumenti sono utilizzati negli addons per Kodi sviluppati da NeverWise.</description>
-    <description lang="en">These toolkit are used in the addons for Kodi developed by NeverWise.</description>
+    <summary lang="it_IT">Strumenti per gli addons audio/video di Kodi.</summary>
```

Comment:
You're right. It's not my code so I missed this change.
I tested it with gotham and it works well (I tested it with the addon submitted to isengard). Anyway if I have to push it to helix isn't a problem since the addon that needs the update is in isengard repo.